lib/msf/core/exploit.rb

```diff
@@ -132,6 +132,9 @@ def initialize(code, reason)
           ''
       end
       super(code, "#{msg} #{reason}".strip, reason)
+
+      # details is intentionally omitted from the struct fields
+      @details = details
```

Comment:
After talking to the team we decided that it's a better idea to move this back into the struct fields.

Reply:
Sensible!